libglusterfs/src/inode.c

```diff
@@ -699,6 +706,8 @@ inode_t *
 inode_new(inode_table_t *table)
 {
     inode_t *inode = NULL;
+    xlator_t *xl = THIS;
+    gf_boolean_t root_with_ref = _gf_false;
 
     if (!table) {
         gf_msg_callingfn(THIS->name, GF_LOG_WARNING, 0,
```

Comment:
```suggestion
        gf_msg_callingfn(xl->name, GF_LOG_WARNING, 0,
```

Reply:
Totally missed this family of optimizations. Fixed.